roles/system/defaults/main.yml

```diff
@@ -12,6 +12,7 @@ proxy_env:
 
 default_packages:
   common:
+    - authconfig
```

Comment:
Are you sure about it ? Starting with Redhat 8 (Rocky 8, Alma 8) `authselect` replace `authconfig`.

Reply:
I encountered the issue on a CentOS 7 container. Maybe we need to start writing dedicated playbooks for each distro?

Reply:
I think he added this package because it's not part of the lxd image.

imo we should add `legacy: authconfig` and `modern: authselect`

Reply:
Indeed I missed that we could do that.